src/index.ts

```diff
@@ -301,13 +299,13 @@ export const plugin: Plugin<[FlexibleContainerOptions?], Root> = (options) => {
    * if the paragraph node has one child (as Text),
    * control whether the node has end marker ":::" or not (check completeness)
    */
-  function analyzeChild(node: Paragraph): AnalyzeResult {
+  function analyzeChild(node: Paragraph, fenceLen: number): AnalyzeResult {
```

Comment:
will receive fence instead of fenceLen, according to proposal of `getOpeningFence`